testing/infra/k8s/base/stack/apm-server.yaml

```diff
@@ -1,31 +1,33 @@
-apiVersion: agent.k8s.elastic.co/v1alpha1
-kind: Agent
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Did you consider using ECK's ApmServer CRD? https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-apm-server.html

I'm not sure that there's much value it in doing so, TBH, since configuration is pretty trivial. I guess hot reloads are faster using a Deployment?

Reply:
TIL!

Thanks for mentioning this! I updated the code! :+1: